src/web/demo.rkt

```diff
@@ -328,20 +328,21 @@
      (response 202 #"Job in progress" (current-seconds) #"text/plain"
                (list (header #"X-Job-Count" (string->bytes/utf-8 (~a (hash-count *jobs*)))))
                (λ (out) (display (apply string-append
-                                        (for/list ([entry (reverse (unbox timeline))])
-                                          (format "Doing ~a\n" (hash-ref entry 'type))))
+                                         (for/list ([entry (reverse (unbox timeline))])
+                                           (format "Doing ~a\n" (hash-ref entry 'type))))
```

Comment:
Please configure your editor not to randomly modify the indentation style. If you're using VS Code you may want to ask @zaneenders how he set it up.